ansible/roles/postgresql_server/tasks/manage_clusters.yml

```diff
@@ -65,6 +65,19 @@
   loop_control:
     label: '{{ item.item }}'
 
+- name: Log directory path
+  file:
+    name: "{{item.log_directory}}"
+    mode: 0755
+    owner: '{{ item.user | d(postgresql_server__user) }}'
+    group: '{{ item.group | d(postgresql_server__group) }}'
+    state: directory
+    recurse: yes
+  with_flattened:
+    - '{{ postgresql_server__clusters }}'
+  when: item.log_directory is abs
```

Comment:
What? `abs`?

Reply:
Absolute path.

Reply:
I recall the problem was that when the postgresql directory is absolute, then these are needed, otherwise, it "fits" into the system nicely